src/math/fft/fft_16.c

```diff
@@ -8,8 +8,60 @@
 #include <sof/audio/format.h>
 #include <sof/common.h>
 #include <sof/math/fft.h>
+
+#ifdef FFT_GENERIC
 #include <sof/audio/coefficients/fft/twiddle_16.h>
 
+/*
+ * Helpers for 16 bit FFT calculation
+ */
+static inline void icomplex16_add(struct icomplex16 *in1, struct icomplex16 *in2,
```

Comment:
I think you could also make these functions compatible with const arguments:
```suggestion
static inline void icomplex16_add(const struct icomplex16 *in1, const struct icomplex16 *in2,
```
Same for all the other ones.

Reply:
Thanks, done.